css/landing_page.css

```diff
@@ -38,3 +38,9 @@
   text-transform: none;
   letter-spacing: normal;
 }
+
+.dropbutton-multiple .dropbutton-action .ajax-progress {
+  position: relative !important;
```

Comment:
We're trying to avoid !important - instead of using it here, could you add more specificity instead?

Reply:
Hey @yeniatencio. This cant be avoided as it has to precede the styling already defined.

Reply:
Resolved